ansible/roles/gitlab/tasks/configure_gitlab-pages.yml

```diff
@@ -110,3 +110,13 @@
   become: True
   become_user: '{{ gitlab_user }}'
   when: gitlab_register_pages_checkout is changed
+
+- name: Setup GitLab Pages default variables
+  template:
+    src: 'etc/default/gitlab-pages.j2'
+    dest: '/etc/default/gitlab-pages'
+    owner: 'root'
+    group: 'root'
+    mode: '0644'
```

Comment:
The file contains secrets, so it should have more strict permissions. I would suggest changing the mode to `0640' and the group to `git` so that GitLab user can read the contents.

Reply:
Done!